basics/a13.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	s1 := fmt.Sprint("sssd-2.0.0-43")
```

Comment:
What is the difference between initialising the var as string and fmt.Sprint ?

Reply:
in this case i wanted to know how fmt.Sprint works, fmt.Sprint returns a string. 
https://golang.org/pkg/fmt/#Sprint  so trying that out.